data_store/controllers/ingest.py

```diff
@@ -503,7 +503,7 @@ def save_submission_file_s3(excel_file: FileStorage, submission_id: str):
         object_name=f"{fund_type}/{str(uuid)}",
         metadata={
             "submission_id": submission_id,
-            "filename": excel_file.filename,
+            "filename": f"{fund_type}-{str(uuid)}.xlsx",
```

Comment:
I think there are a few other options that might be better for fund teams:

* Sanitise the string by removing anything non-ascii
* Insert more useful data - submission datetime, organisation name, submission ID (rather than UUID), etc?

Reply:
Fund teams are not affected by this change. This is only what AWS stores as a filename.

Fund team get the pretty human readable filename that causes other Sentry errors currently. :laughing: 
Example: `File name: 2024-11-12-td-worcester-city-council-apr2024-sep2024.xlsx`

Reply:
Hm you're right. The only time the filename metadata is used is in admin actions. Do we even need to store the original filename? 🤔

Reply:
We can have a quick chat tomorrow. 

I considered 1st to strip "illegal" chars. But then if this file name affects nothing else, then this is a quick, simple fix.

The user submitted file name still gets echoed in two place, but it's never store, only used during the request:
- HTML template green success box 
- User confirmation email 

This changes only the filename AWS stores.

The file name the Fund Team confirmation email gets is the pretty human readable version. (Until we fix the problem with old fund team confirmation email file name error.)

Reply:
I think as it's also for audit purposes can we keep the original filename but make it us-ascii safe?